### PR TITLE
IPD and Sensor Scaling Support

### PR DESCRIPTION
**Pull request by hyperlogic**
NO NOT MERGE, this PR is to create a demo build.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2678/HighFidelity-Beta-PR11234-d62a0ea8ff7e1c0e63586ec104ed37570fc5b587.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2678/HighFidelity-Beta-PR11234-d62a0ea8ff7e1c0e63586ec104ed37570fc5b587.dmg

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -2497,9 +2504,13 @@ void Application::paintGL() {
...
-            auto baseProjection = renderArgs.getViewFrustum().getProjection();
-            auto hmdInterface = DependencyManager::get<HMDScriptingInterface>();
-            float IPDScale = hmdInterface->getIPDScale();
+            mat4 eyeProjections[2];
+
+            // adjust near clip plane by heightRatio
+            auto baseProjection = glm::perspective(renderArgs.getViewFrustum().getFieldOfView(),
+                                                   renderArgs.getViewFrustum().getAspectRatio(),
+                                                   renderArgs.getViewFrustum().getNearClip() * sensorToWorldScale,
+                                                   renderArgs.getViewFrustum().getFarClip());
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: why aren't you also adjusting the far clip?

Comment (hyperlogic) [inline interface/src/Application.cpp]: Adjusting the far clip will cause large far away objects to get clipped, which might be noticed visually.  Only adjusting the near clip plane means less resolution for the z-buffer, but after talking with @samcake, the 24-bit z-buffer should have sufficient resolution.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2731/HighFidelity-Beta-PR11234-246097b65ebb2b10aef3d657a559f34858b27570.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2731/HighFidelity-Beta-PR11234-246097b65ebb2b10aef3d657a559f34858b27570.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2825/HighFidelity-Beta-PR11234-3ac09b7716f2da9dc80031df14e3a24376be0977.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2825/HighFidelity-Beta-PR11234-3ac09b7716f2da9dc80031df14e3a24376be0977.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2828/HighFidelity-Beta-PR11234-937f4005342a82b08b0536b79757784d03897057.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2828/HighFidelity-Beta-PR11234-937f4005342a82b08b0536b79757784d03897057.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2838/HighFidelity-Beta-PR11234-573985c7c96497580d41798406c2bc37ab1a927b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2838/HighFidelity-Beta-PR11234-573985c7c96497580d41798406c2bc37ab1a927b.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2844/HighFidelity-Beta-PR11234-5230b03f4f73ad7846fd27b0a59630cf44aa1946.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2844/HighFidelity-Beta-PR11234-5230b03f4f73ad7846fd27b0a59630cf44aa1946.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2895/HighFidelity-Beta-PR11234-9cd8b6991ed5f87d3ffae3afcca5b4c97b0920ad.exe

**Comment by cozza13** [comment]
ok to test

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2897/HighFidelity-Beta-PR11234-9cd8b6991ed5f87d3ffae3afcca5b4c97b0920ad.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2897/HighFidelity-Beta-PR11234-9cd8b6991ed5f87d3ffae3afcca5b4c97b0920ad.exe

**Comment by Nex-Pro** [comment]
Using windows build ending in 920ad

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2945/HighFidelity-Beta-PR11234-6e6687c3a62fe5490e5f09476d14535ce921fc16.dmg

**Comment by marko8904** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2947/HighFidelity-Beta-PR11234-6e6687c3a62fe5490e5f09476d14535ce921fc16.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2947/HighFidelity-Beta-PR11234-6e6687c3a62fe5490e5f09476d14535ce921fc16.exe

**Comment by marko8904** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2964/HighFidelity-Beta-PR11234-9c6b8a6826d11f1c7287d9bd764596beb5702233.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2979/HighFidelity-Beta-PR11234-a0eb8152a6636e5d4a78aad9493f40f2d87a21ef.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2979/HighFidelity-Beta-PR11234-a0eb8152a6636e5d4a78aad9493f40f2d87a21ef.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3006/HighFidelity-Beta-PR11234-a1b94fd9d269b3bc215099296f24ef1c0585d0bc.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3006/HighFidelity-Beta-PR11234-a1b94fd9d269b3bc215099296f24ef1c0585d0bc.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3025/HighFidelity-Beta-PR11234-26e9e6e580fb7497c013ae75cc784f9a4763deb4.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3025/HighFidelity-Beta-PR11234-26e9e6e580fb7497c013ae75cc784f9a4763deb4.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3052/HighFidelity-Beta-PR11234-f0912987a384dc33c289c4f00718b5c7b4816513.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3052/HighFidelity-Beta-PR11234-f0912987a384dc33c289c4f00718b5c7b4816513.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3064/HighFidelity-Beta-PR11234-23ff1b9eade583752416300186616ccc4fd8383c.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3065/HighFidelity-Beta-PR11234-3d3bb2853ac98fdf0aedc8ad59f65e61d42705b6.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3065/HighFidelity-Beta-PR11234-3d3bb2853ac98fdf0aedc8ad59f65e61d42705b6.exe

**Comment by anshuman64** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3070/HighFidelity-Beta-PR11234-74f974c68ff2aa3a1763355c36e57ae56424c7e4.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3073/HighFidelity-Beta-PR11234-af8b147321169c6617f8933e7cc43d7f6f01b8d5.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3073/HighFidelity-Beta-PR11234-af8b147321169c6617f8933e7cc43d7f6f01b8d5.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3087/HighFidelity-Beta-PR11234-a5553d5e1c4182344abff421b1404c37e406c2aa.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3087/HighFidelity-Beta-PR11234-a5553d5e1c4182344abff421b1404c37e406c2aa.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3093/HighFidelity-Beta-PR11234-2f850826f2b36395f98ed1b3765e8607f096ec44.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3095/HighFidelity-Beta-PR11234-37f959d74975f29dd62ab7d0f81c30e1f2c50a77.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3095/HighFidelity-Beta-PR11234-37f959d74975f29dd62ab7d0f81c30e1f2c50a77.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3109/HighFidelity-Beta-PR11234-3ff707f27eada1487e62bc19dd8e070083011bd0.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3109/HighFidelity-Beta-PR11234-3ff707f27eada1487e62bc19dd8e070083011bd0.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3114/HighFidelity-Beta-PR11234-4c302b899de3081fb76b12703a370517984f3e87.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3114/HighFidelity-Beta-PR11234-4c302b899de3081fb76b12703a370517984f3e87.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3119/HighFidelity-Beta-PR11234-94e26a554d85e4d5fb82bcba67db3b711ca86f30.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3119/HighFidelity-Beta-PR11234-94e26a554d85e4d5fb82bcba67db3b711ca86f30.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3122/HighFidelity-Beta-PR11234-d11a749365e28453d5e8dafaa5e82d3b60e68107.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3122/HighFidelity-Beta-PR11234-d11a749365e28453d5e8dafaa5e82d3b60e68107.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3135/HighFidelity-Beta-PR11234-f4f9c29bffacb6e02961b06bae60046a32b04e7d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3137/HighFidelity-Beta-PR11234-7ca498dc7f818de00c5875755856460aaa579d92.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3137/HighFidelity-Beta-PR11234-7ca498dc7f818de00c5875755856460aaa579d92.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3149/HighFidelity-Beta-PR11234-fde5e14c045466a33db629654556219e9fd09864.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3149/HighFidelity-Beta-PR11234-fde5e14c045466a33db629654556219e9fd09864.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3150/HighFidelity-Beta-PR11234-10e29d1cf0802fa9a502f25e4fbe43e0f487ddd9.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3150/HighFidelity-Beta-PR11234-10e29d1cf0802fa9a502f25e4fbe43e0f487ddd9.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3152/HighFidelity-Beta-PR11234-758f049f306b9f56e522077032733ba89289b881.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3152/HighFidelity-Beta-PR11234-758f049f306b9f56e522077032733ba89289b881.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3156/HighFidelity-Beta-PR11234-e453bf8d559fdf59dee32f9ddf5a22547f25c1c4.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3156/HighFidelity-Beta-PR11234-e453bf8d559fdf59dee32f9ddf5a22547f25c1c4.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3177/HighFidelity-Beta-PR11234-72e0c844e47c197983bec3623772ea97af233629.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3177/HighFidelity-Beta-PR11234-72e0c844e47c197983bec3623772ea97af233629.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3211/HighFidelity-Beta-PR11234-b81f85e1ae002880eb416d4601844d970f6a5121.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3211/HighFidelity-Beta-PR11234-b81f85e1ae002880eb416d4601844d970f6a5121.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3215/HighFidelity-Beta-PR11234-bb5e808051ce3e44f9d579bf1f49d2d483afada1.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3215/HighFidelity-Beta-PR11234-bb5e808051ce3e44f9d579bf1f49d2d483afada1.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3231/HighFidelity-Beta-PR11234-5a918a3033bfd63291609f3ebf250752e43239f0.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3238/HighFidelity-Beta-PR11234-cc7bb9a1bf9d2d1afefc0a13c72c64320a032c60.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3238/HighFidelity-Beta-PR11234-cc7bb9a1bf9d2d1afefc0a13c72c64320a032c60.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3265/HighFidelity-Beta-PR11234-9a1fa9b2f29e7a6e514c1f6d4f686dd202ab8d39.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3265/HighFidelity-Beta-PR11234-9a1fa9b2f29e7a6e514c1f6d4f686dd202ab8d39.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3274/HighFidelity-Beta-PR11234-de5f0ad987acaecbe1c34429394f751e6193d201.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3274/HighFidelity-Beta-PR11234-de5f0ad987acaecbe1c34429394f751e6193d201.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3287/HighFidelity-Beta-PR11234-a505a16232430fc0a158d2b0e74b5b699c525c7d.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3287/HighFidelity-Beta-PR11234-a505a16232430fc0a158d2b0e74b5b699c525c7d.dmg

**Comment by hyperlogic** [comment]
@anshuman64 will update this branch with a QA Ready tag and a QA Test plan

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3308/HighFidelity-Beta-PR11234-7c725756f1ee87f99c051f518d55dbe9a340edb6.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3308/HighFidelity-Beta-PR11234-7c725756f1ee87f99c051f518d55dbe9a340edb6.dmg

**Comment by hyperlogic** [comment]
building again, after s3 public folder is back up.  build this please

**Comment by anshuman64** [comment]
Test Plan

Go through the tests in the Google Doc below with the following preconditions: 

1. Login
2. Go to location wikiplanet/newyork
3. Open Marketplace. Search "Tiny Skeleton". Use that avatar.

https://docs.google.com/spreadsheets/d/1BoR9dKcOtBeUdE09ya2vN6h6nJilsu-5O40oOn35WSY/edit#gid=1968671527

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3321/HighFidelity-Beta-PR11234-7959988622b0123d6087b179b60a1754294cb544.dmg

**Comment by sethalves** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3330/HighFidelity-Beta-PR11234-7959988622b0123d6087b179b60a1754294cb544.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3330/HighFidelity-Beta-PR11234-7959988622b0123d6087b179b60a1754294cb544.exe

File: scripts/system/notifications.js [hunk 2]
```diff
@@ -242,14 +242,14 @@
...
-
+        var sensorScaleFactor = MyAvatar.sensorToWorldScale;
         if (isOnHMD) {
             // Calculate 3D values from 2D overlay properties.
 
             noticeWidth = notice.width * NOTIFICATION_3D_SCALE + NOTIFICATION_3D_BUTTON_WIDTH;
             noticeHeight = notice.height * NOTIFICATION_3D_SCALE;
 
-            notice.size = { x: noticeWidth, y: noticeHeight };
+            notice.size = { x: noticeWidth, y: noticeHeight};
```
Comment (sethalves) [inline scripts/system/notifications.js]: space before } should have stayed

**Comment by hyperlogic** [comment]
weird build failure try again

    20:37:15 physics.lib(ObjectActionOffset.obj) : fatal error LNK1318: Unexpected PDB error; RPC (23) '(0x000006BA)' [C:\jenkins\workspace\pr-build\label\windows\build\interface\interface.vcxproj]

build this please

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3337/HighFidelity-Beta-PR11234-f534ee5ad935d6b513de03068d4932b98d805d04.exe

File: scripts/system/controllers/controllerDisplay.js [hunk 3]
```diff
@@ -59,14 +59,16 @@ createControllerDisplay = function(config) {
         },
 
         setPartVisible: function(partName, visible) {
-            return;
+            // Disabled
+            /*
```
[the remark this answers is not in the document]
Comment (hyperlogic) [reply scripts/system/controllers/controllerDisplay.js]: No, this was disabled on master as well.

File: interface/src/Application.cpp [hunk 4]
```diff
@@ -6648,7 +6667,7 @@ void Application::addAssetToWorldCheckModelSize() {
         if (dimensions != DEFAULT_DIMENSIONS) {
 
             // Scale model so that its maximum is exactly specific size.
-            const float MAXIMUM_DIMENSION = 1.0f;
+            const float MAXIMUM_DIMENSION = 1.0f * getMyAvatar()->getSensorToWorldScale();
```
[the remark this answers is not in the document]
Comment (hyperlogic) [reply interface/src/Application.cpp]: ok, will fix on secondary IPD branch

File: scripts/system/notifications.js (hunk 2 above)
Comment (hyperlogic) [reply scripts/system/notifications.js]: will fix on secondary ipd branch

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3337/HighFidelity-Beta-PR11234-f534ee5ad935d6b513de03068d4932b98d805d04.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3341/HighFidelity-Beta-PR11234-f534ee5ad935d6b513de03068d4932b98d805d04.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3393/HighFidelity-Beta-PR11234-7e2b2960ed26cfc84da2a2051f7a444300d8e2da.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3393/HighFidelity-Beta-PR11234-7e2b2960ed26cfc84da2a2051f7a444300d8e2da.exe

**Comment by anshuman64** [comment]
Known issues: https://highfidelity.fogbugz.com/f/search/?sColumns=Category-Favorite-Case-TitleComment-Status-AssignedTo-Priority-Area-Tags-OpenedBy-Milestone&sSearchFor=tag%3A%22IPDtasks%22&sSorts=Priority

**Pull request merged**